ODIN/chain.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "odin-mainnet-freya",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "OdinChain",
+  "chain_id": "odin-mainnet-freya",
+  "bech32_prefix": "odin",
+  "daemon_name": "odind",
+  "node_home": "$HOME/.odin",
+  "genesis": "https://github.com/ODIN-PROTOCOL/networks/blob/master/mainnets/odin-mainnet-freya/final_genesis.json",
+  "key_algos": [
+    "secp256k1"`	
+  ],
+  "slip44": 494,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "loki",
+        "fixed_min_gas_price": 0.0125
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/ODIN-PROTOCOL/networks/blob/master/mainnets/odin-mainnet-freya",
+    "recommended_version": "v0.2.0",
+    "binaries": {
+      "linux/amd64": "https://github.com/GeoDB-Limited/odin-core/releases/download/v0.2.0/odinprotocol_v0.2.0_amd64.deb"
+    },
+    "compatible_versions": [
+      "v0.2.0"
+    ]
+  },
+  "peers": {
+    "persistent_peers": [
+      {
+        "id": "9d16b1ce74a34b869d69ad5fe34eaca614a36ecd",
+        "address": "35.241.238.207:26656",
+        "provider": "Odin Protocol"
+      },
+      {
+        "id": "02e905f49e1b869f55ad010979931b542302a9e6",
+        "address": "35.241.221.154:26656",
+        "provider": "Odin Protocol"
+      },
+      {
+        "id": "4847c79f1601d24d3605278a0183d416a99aa093",
+        "address": "34.140.252.7:26656",
+        "provider": "Odin Protocol"
+      },
+      {
+        "id": "0165cd0d60549a37abb00b6acc8227a54609c648",
+        "address": "34.79.179.216:26656",
+        "provider": "Odin Protocol"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "http://34.79.179.216:26657",
+        "provider": "Odin Protocol"
+      }
+    ],
+    "rest": [
+      {
+        "address": "http://34.79.179.216:1317/",
+        "provider": "Odin Protocol"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "odin web",
+      "url": "https://odin-freya-website.web.app/",
```

Comment:
This explorer doesn't seem to be loading for me.  I get an "Initializing Failed" error.

Reply:
Also, it seems that this is the explorer, no?

https://odin-scan.web.app/